### DISCUSSION TITLE
Tailwind Styles Not Being Applied. PRODUCTION ONLY

### DISCUSSION THREAD

**Discussion by jtoomay** [vercel/community#5055] [Help] [answered]
### Page to Investigate

https://next-portfolio-swart-three.vercel.app/

### Steps to Reproduce

For some reason, some Tailwind styles that I am adding are not being applied when I push to production only. The app works flawlessly on my local environment, but when I push to Netfliy, the Navigation is broken. It gets some styles like "p-4", but the main styles that I have noticed that are not being applied are "w-80", "top-0", "left-0", "bottom-0". 

I'm not sure why this is only happening on Vercel. I will leave a link to the repo below and you can see how it works fine when the app is run locally. 

Has anyone else ran into this? Its possible I am making a noob mistake since I don't have much experience making next.js applications

NOTE: please don't clown me for the code. it is in an early state and I am in the process of learning server actions lol

VERCEL: https://next-portfolio-swart-three.vercel.app/
GITHUB REPO:  https://github.com/jtoomay/next-portfolio

**Comment by amyegan** [comment]
I'm not able to run your project myself to see what it's supposed to look like. The deployment you shared does seem to have some CSS styling applied, but I'm not sure how it's actually supposed to look.

Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that doesn't require any private environment variables? Some screenshots to demonstrate the differences between localhost and production would also help me understand what looks wrong 🙏

**Reply by jtoomay** [reply]
Sorry for the late response. I will upload some screenshots demonstrating the issue. 

This is on my local environment. This is with the navigation menu open. It better demonstrates the problem. Notice the styles that are underlined. This is working as expected. 
![image](https://github.com/vercel/community/assets/81326775/6b1be682-cccc-49b3-becb-f45779234626)
<br /> 

The trouble comes when we push to Vercel. This is what it looks like on Vercel's environment that you can see visiting the link above. 
![image](https://github.com/vercel/community/assets/81326775/b9394eea-0e23-42e9-a995-a9cbeb170164)
<br /> 
Notice the styles in the class list that are not being used. p-4 and fixed were the only styles that were actually applied. I don't understand why this is only happening to me on production. Have you ever seen something like this? Let me know if you need anything else. Thanks for looking into this!

**Comment by jtoomay** [comment, marked as answer]
At last! I have found the solution. My folder that contained the navigation was called "Components". The folder path that was in my twConfig was "components" (lowercase). When I updated the path to "Components", it worked flawlessly. Hope this helps someone if you are going through a similar issue
